packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1525,6 +1525,24 @@ mixin WidgetInspectorService {
     return false;
   }
 
+  /// Notify debuggers to open an inspector on the object.
+  void _sendInspectEvent(Object? object){
+    inspect(object);
+
+    final _Location? location = WidgetInspectorService.instance._getSelectedSummaryWidgetLocation(null, 'my-group');
```

Comment:
avoid passing in a bogus group name.  That can lead to leaking memory if you never free the references retained by the group. In this case I think the group name is never needed so you are ok but it got accidentally passed around places it wasn't used.